horace_core/sqw/@DnDBase/private/init_.m

```diff
@@ -128,8 +139,9 @@
         error(['HORACE:', class(obj),':invalid_argument'], ...
             'Unrecognised number or type of the input arguments')
     end
-
 else
     error(['HORACE:', class(obj),':invalid_argument'], ...
         'unknown input for %s constructor',class(obj));
 end
+
+function ob
```

Comment:
looks like this hasn't been cleaned up

Reply:
yes, I have apparently forgot to  committ the clean-up